templates/user/dashboard/feeds.tmpl

```diff
@@ -86,6 +98,18 @@
 						<p class="text light grey has-emoji">{{index .GetIssueInfos 1}}</p>
 					{{else if (or (or (eq .GetOpType 12) (eq .GetOpType 13)) (or (eq .GetOpType 14) (eq .GetOpType 15)))}}
 						<span class="text truncate issue title has-emoji">{{.GetIssueTitle}}</span>
+					{{else if eq .GetOpType 23}}
+						<span class="text truncate issue title has-emoji">{{.GetIssueTitle}}</span>
+            <p class="text light grey has-emoji">{{if .HasIssueRemovedLabel}}<a class="ui label" href="{{.GetRepoLink}}/issues?labels={{.GetIssueRemovedLabelID}}" style="color: {{.GetIssueRemovedLabelForegroundColor}}; background-color: {{.GetIssueRemovedLabelColor}}; text-decoration: line-through">{{.GetIssueRemovedLabelName | Sanitize}}</a>{{end}} {{if .HasIssueAddedLabel}}<a class="ui label" href="{{.GetRepoLink}}/issues?labels={{.GetIssueAddedLabelID}}" style="color: {{.GetIssueAddedLabelForegroundColor}}; background-color: {{.GetIssueAddedLabelColor}}">{{.GetIssueAddedLabelName | Sanitize}}</a>{{end}}</p>
```

Comment:
The only thing I concerned about is in this one single feed, could cause multiple DB queries. How about save the information (i.e. label ID, color, name) to the at creation of the feed (`Action`, use `|` to split). This way, we could basically avoid most of DB queries for displaying the feeds.

The feed content is meant to/could be in the state of immutable, so store the instant state of information at creation sounds better to me.

What do you think?